spring-boot/current-community/istio-distributed-tracing/booster.yaml

```diff
@@ -0,0 +1,6 @@
+name: Istio - Spring Boot - Distributed Tracing
+description: Booster to demonstrate how Istio's service mesh and Distributed Tracing - Jaeger interact.
+source:
```

Comment:
No `runsOn` tag here?

Reply:
No idea how the Istio boosters need to be qualified, actually… :(

Reply:
They should be tested on the clusters that this booster will run: `pro`, `starter`or `osio` or simply `none` if it should be available as a zip file only.

Reply:
what's the name for OCP then?

I believe the Istio boosters are only intended to run on a local OCP install

Reply:
Ah yes, that would be `local`. I need to put that information in the README

Reply:
Just to make sure, if we only want the Istio boosters torun on local OCP install, we should have:
```yaml
metadata:
  runsOn: 
    - 'local'
```

Reply:
Yes, that seems to be correct. Make sure to test locally before pushing ;)